rtcstats.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 (function() {
+  // transforms a maplike to an object. Mostly for getStats +
+  // JSON.parse(JSON.stringify())
+  function m2o(m) {
```

Comment:
i would not assume compilers or minifiers here, this thing is intended to be used via script tags